FLT/Deformations/IsResidueAlgebra.lean

```diff
@@ -0,0 +1,116 @@
+import Mathlib.RingTheory.LocalRing.ResidueField.Basic
+import Mathlib.Logic.Function.Defs
+import FLT.Mathlib.RingTheory.LocalRing.Defs
+
+open Function
+open scoped TensorProduct
+
+namespace Deformation
+
+variable (𝓞 : Type*)
+  [CommRing 𝓞] [IsLocalRing 𝓞]
+
+local notation3:max "𝓴" 𝓞 => (IsLocalRing.ResidueField 𝓞)
+
+variable (A : Type*) [CommRing A] [Algebra 𝓞 A] [IsLocalRing A] [IsLocalHom (algebraMap 𝓞 A)]
+
+abbrev modMap_high : 𝓞 →+* 𝓴 A :=
+  (IsLocalRing.residue A).comp (algebraMap 𝓞 A)
+
+abbrev modMap : (𝓴 𝓞) →+* 𝓴 A :=
+  IsLocalRing.ResidueField.map (algebraMap 𝓞 A)
```

Comment:
I think you should just use `algebraMap 𝓞 (𝓴 A)` and `algebraMap (𝓴 𝓞) (𝓴 A)` and not add new abbrevs.